source/common/http/filter_manager.h

```diff
@@ -1016,6 +1013,7 @@ class FilterManager : public ScopeTrackedObject,
     ActiveStreamDecoderFilter* latest_data_decoding_filter_{};
   };
 
+protected:
```

Comment:
Does this make state_ protected instead of private? If so, can we avoid that in some way? Protected members are not really style guide kosher.

Reply:
fixed